pkg/rehearse/jobs.go

```diff
@@ -220,6 +222,8 @@ func NewExecutor(toBeRehearsed map[string][]prowconfig.Presubmit, prNumber int,
 	dryRun bool, loggers Loggers, pjclient pj.ProwJobInterface) *Executor {
 	rehearsals := configureRehearsalJobs(toBeRehearsed, prRepo, prNumber, loggers)
 	return &Executor{
+		RehearsalJobCount: len(rehearsals),
+
```

Comment:
nit: no need for the extra line here

Reply:
I wanted to separate private and exported fields